### ISSUE TITLE
Before dying, humans doing a random animation when shot in the head.

### ISSUE THREAD

**Issue by Nicovw321** [studio-minus/ppg-bugs-and-requests#2540]
### Feature description

It's a new behaviour for humans, when they get shot in the front of their head with for example, a glock, they start doing a random animation like if their brain misfired before dying completely.

### Feature purpose

It would make the humans more realistic and satisfying to shoot them in the head, don't ask why it is satisfying.

**Comment by bibbybabby** [comment]
bruh what the hell man they supposed to do the griddy befory dying lmao?? its like one of those indian movies lol

**Comment by Drogo-man** [comment]
Humans gettin' sturdy.

**Comment by Nicovw321** [comment]
> bruh what the hell man they supposed to do the griddy befory dying lmao?? its like one of those indian movies lol

yes! make them do something, just becoming a gmod ragdoll is not fun, at least when they survive a shot in the head it's fun

**Comment by Nicovw321** [comment]
> Humans gettin' sturdy.

sturdy than me yes, and man they don't get sturdy at all just fall and not react
